app.rb

```diff
@@ -1,8 +1,30 @@
-require 'sinatra/base'
+require "sinatra"
+require "sinatra/base"
+require "./lib/peep"
```

Comment:
This looks great on the whole and clearly demonstrates some of the week's learning objectvives. The one thing I would recommend is that you revisit RESTful routing.